.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         # Test all supported versions on Ubuntu:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, ubuntu-24.04-arm]
         python: ["3.9", "3.10", "3.11", "3.12", "3.13"]
         experimental: [false]
         build: ['']
```

Comment:
Do we want to include some experimental tasks on arm as well?

Reply:
yeah we should add!